libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -18,12 +15,28 @@ use types_rs::{election::UnitIntervalValue, geometry::IntersectionBounds};
 use crate::{
     ballot_card::{Geometry, Orientation},
     debug::{self, draw_timing_mark_debug_image_mut, ImageDebugWriter},
-    image_utils::{expand_image, match_template, WHITE},
+    image_utils::{expand_image, match_template, Inset, WHITE},
     interpret::{self, Error},
     qr_code_metadata::BallotPageQrCodeMetadata,
     timing_mark_metadata::BallotPageTimingMarkMetadata,
 };
 
+pub struct BallotImage {
+    pub image: GrayImage,
+    pub threshold: u8,
+    pub border_inset: Inset,
+}
+pub struct BallotPage {
+    pub ballot_image: BallotImage,
+    pub geometry: Geometry,
+}
+
+pub struct BallotCard {
+    pub side_a: BallotImage,
+    pub side_b: BallotImage,
+    pub geometry: Geometry,
+}
```

Comment:
Maybe these could live in the `ballot_card` module? That seems slightly more thematically appropriate